### ISSUE TITLE
Parallelize UI test execution

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/focus-android#3638]
### Why/User Benefit/User Problem
Currently our UI test execution is slow due to serial execution and monitoring results which require a painful drilldown into TaskCluster logs and Firebase.  For these reasons, we restrict our UI test execution to master only, which further removes developers from monitoring/fixing test failures. Implementing the Flank library in our CI will enable us to shard test execution and will output a variety of test results locally.  There are pending issues to consolidate the test configurations and also be able to push issues directly into GCP storage.

### What / Requirements
Update /tools/taskcluster scripts to execute using flank jar (rather than directly from Firebase API)

### Acceptance Criteria (how do I know when I’m done?)
Tests execute in parallel.
NOTE:  email notification using taskcluster notify will be separate PR
